### ISSUE TITLE
Public release acknowlegment

### ISSUE THREAD

**Issue by ajkiessl** [psu-libraries/etda_workflow#823]
Add another field to the acknowledgment form for users to agree that their submission will be publicly released.  This field should be the last acknowledgment on that page.

Wording from the Grad School:

For Doctoral students:

>> "I acknowledge that my approved dissertation will be publicly released through the University Libraries as part of the Penn State Graduate Council's Ph.D. degree requirements."

 For Masters' students:

>> "I acknowledge that my approved thesis will be publicly released through the University Libraries as part of the Penn State Graduate Council's master's degree requirements."

I need to confirm that there will be different wording for students with confidential holds.  For now we know that we want to convey this to those with confidential holds:

>> i) public release of a thesis or dissertation still occurs when a student has a Confidentiality Hold
>> ii) if the student has any questions, that they contact the Office of Theses and Dissertations in the Graduate School at gradthesis@psu.edu, providing the opportunity for students with ongoing concerns to discuss their situations with the Graduate School. In well-supported cases, the Graduate School would embargo the thesis or dissertation for 2 years (similar to the intellectual property embargo)

**Comment by Smullz622** [comment]
@ajkiessl should that note have a signature with it as well?

**Comment by ajkiessl** [comment]
@Smullz622 No, I was thinking it would be in between the acknowledgment statement and the signature box
